pkg/azuredisk/azuredisk_v2.go

```diff
@@ -112,6 +140,10 @@ type DriverV2 struct {
 	kubeClientQPS                     int
 }
 
+func init() {
+	clientmetrics.RateLimiterLatency = &LatencyAdapter{metric: rateLimiterLatency}
```

Comment:
We should call the `[Register](https://pkg.go.dev/k8s.io/client-go@v0.24.2/tools/metrics#Register)` function to override the default metrics reporter. It can only be called once, it doesn't appear that anything does so currently. I expected this is the supported way for users to provide metrics loggers.

Reply:
`Register` is already being called in controller-runtime in an [`init()`](https://github.com/kubernetes-sigs/controller-runtime/blob/ffd9ec8768b7d2e82aee6b0c76d481a5bdda4075/pkg/metrics/client_go_adapter.go#L139) function (this package is imported by the manager package) before we even get to our code to update any of the options, so calling it again here will have no effect. The only way we can change the configuration and enable the rate limiter in our code is by updating the package variables in client-go.